### PR TITLE
update path poc

### PR DESCRIPTION
**Pull request by odra**
# Digger update path

## Changes

**Parameters**
- ANDROID_SDK_AUTO_CONFIG: boolean blag to install the default packages defined in digger apb
- ANDROID_SDK_CONFIG: custom config string to be used when installing packages (ignored if `ANDROID_SDK_AUTO_CONFIG` is set to true)

**Playbooks**

- A new default.cfg file to be used when ANDROID_SDK_AUTO_CONFIG is set to true
- New update playbook and role

**Verification steps:**

- Provision digger apb with the "ANDROID_SDK_AUTO_CONFIG" field checked - it should deploy the default packages described in the default.cfg file
- Uncheck "ANDROID_SDK_AUTO_CONFIG" and add the following config string in "ANDROID_SDK_CONFIG": https://github.com/aerogear/androidctl/blob/master/examples/android.cfg
- Provision the apb without both vars and update it by adding the above config config string into the  `ANDROID_SDK_CONFIG` field

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,31 @@ ICBwYXR0ZXJuOiBeeWVzJAogICAgICB0eXBlOiBzdHJpbmcKICAgICAgZGVzY3JpcHRpb246ICJU\
...
+ICAgICAgZGlzcGxheV90eXBlOiBjaGVja2JveAogICAgICBkZXNjcmlwdGlvbjogIkluc3RhbGxz\
+IHRoZSBkZWZhdWx0IGFuZHJvaWQgc2RrIHBhY2thZ2VzIgogICAgICB0aXRsZTogQW5kcm9pZCBT\
+REsgQXV0byBJbnN0YWxsCiAgICAgIHJlcXVpcmVkOiBGYWxzZQogICAgICBkaXNwbGF5X2dyb3Vw\
+OiBBbmRyb2lkIFNESwogICAgLSBuYW1lOiBBTkRST0lEX1NES19DT05GSUcKICAgICAgdHlwZTog\
+c3RyaW5nCiAgICAgIGRpc3BsYXlfdHlwZTogdGV4dGFyZWEKICAgICAgZGVzY3JpcHRpb246ICJB\
+bmRyb2lkIFNESyBjb25maWcgZmlsZSBjb250ZW50IgogICAgICB0aXRsZTogQW5kcm9pZCBTREsg\
+Y29uZmlnCiAgICAgIHJlcXVpcmVkOiBGYWxzZQogICAgICB1cGRhdGFibGU6IFRydWUKICAgICAg\
+ZGlzcGxheV9ncm91cDogQW5kcm9pZCBTREsK"
+
+
```
Comment (ghost) [inline Dockerfile]: empty lines should be removed

File: roles/provision-buildfarm-apb/files/default.cfg [hunk 2]
```diff
@@ -0,0 +1,53 @@
+[base]
```
Comment (ghost) [inline roles/provision-buildfarm-apb/files/default.cfg:1]: nice!, wanted also move it to file

**Review by vchepeli — CHANGES_REQUESTED** [review]

File: apb.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@ async: optional
 tags:
   - mobile-service
 metadata:
-  displayName: Mobile CI/CD
+  displayName: Mobile CI/CD (dev)
```
Comment (maleck13) [inline apb.yml]: why is it name (dev)?

Comment (odra) [reply apb.yml]: forgot to remove that, I use this name when I am testing the apb

**Review by mikenairn — APPROVED** [review]
Verified by executing a number of scenarios, more info here https://issues.jboss.org/browse/AEROGEAR-2500?focusedCommentId=13573420&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13573420
Functionally it did all work as expected, so will approve, but did notice a few usability issues that might want looked at:

* Different android config format during initial provision that during an upgrade via an edit of the service config.
* Not really clear what either of the 2 provision configuration options do or what format the config should be in.
* If the SDK provision fails, it doesn't make that very clear or easy to figure out why.

**Comment by odra** [comment]
We will cover the debug/cfg format with docs and the other issues are being worked on their respective upstream repositories.

**Pull request merged**